src/lib/Telemetry/telemetry.cpp

```diff
@@ -0,0 +1,203 @@
+#include <cstdint>
+#include <cstring>
+#include <telemetry.h>
+#include <crsf_protocol.h>
+
+#if defined(UNIT_TEST)
+#include <iostream>
+using namespace std;
+#endif
+
+#if defined(PLATFORM_ESP8266) || defined(TARGET_R9M_RX) || defined(UNIT_TEST)
+
+Telemetry::Telemetry()
+{
+    telemtry_state = TELEMTRY_IDLE;
```

Comment:
```suggestion
    telemetry_state = TELEMETRY_IDLE;
```

Any particular reason for this typo? Might just be missing something here in the bigger picture.

Reply:
Should be able to just "find & replace" all places